controllers/tasksRequests.js

```diff
@@ -43,6 +46,73 @@ const fetchTaskRequestById = async (req, res) => {
   }
 };
 
+const addTaskRequests = async (req, res) => {
+  try {
+    const taskRequestData = req.body;
+    const usernamePromise = usersUtils.getUsername(taskRequestData.userId);
+    if (req.userData.id !== taskRequestData.userId && !req.userData.roles?.super_user) {
+      return res.boom.forbidden("Not authorized to create the request");
+    }
```

Comment:
req.userData.id !== taskRequestData.userId

what above condition checking??

Reply:
- `req.userData.id` is the id of logged in user.
- `taskRequestData.userId` is the user id we receive in req body.

We are allowing only logged in user and super users to create a request.